aospy/test/test_roundtrip.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+02-04-2016 SKC: Set of end-to-end tests of aospy.  Currently set up to test for
```

Comment:
You can omit the date and initials...I know I started us down that path a while back, but now I've realized that Github's blame makes this unnecessary (and the more discussion-oriented comments I use to include are much better, I now realize, as Github issues).